Author: George Kiagiadakis <kiagiadakis.george@gmail.com>
Description: Do not search for KDE4Workspace, since it is not used.
 Requires patch 01_no_useless_linking.diff to be applied too.
Forwarded: no

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,4 @@
 find_package(KDE4 REQUIRED)
-find_package(KDE4Workspace REQUIRED)
 include(KDE4Defaults)
 include(MacroOptionalAddSubdirectory)
 
